src/components/sideNav.js

```diff
@@ -21,11 +24,21 @@ function formatMenu(input) {
         } else {
           menu[type] = [race]
         }
+        option.options.push({ label: race.Title, value: race.fields.slug })
       }
     })
+    if (option.options.length) {
+      options.push(option)
```

Comment:
I'm a little lost with all the options...

Reply:
I'll change the variable name to something like menuGroup